src/utils/path.ts

```diff
@@ -43,6 +43,10 @@ export async function resolveImportPath(
   currentDirPath: string,
   pathAlias: PathAlias
 ): Promise<string> {
+  if (!moduleName) {
```

Comment:
Which case will cause the empty module name? And I think this would better be checked in `getClickInfo`.

Reply:
Sorry, forgot the special bad case, remove it now. We will fix it when we re encount same case.